frontend/src/components/types/ProfileInterface.ts

```diff
@@ -0,0 +1,24 @@
+export interface ProfileInterface {
+    __v: string;
+    _id: string;
+    userID: string;
+    profileName: string;
+    about: string;
+    profilePicture: string;
+    coverPicture: string;
+    websiteUrl: string;
+    instagramUrl: string;
+    twitterUrl: string;
+    facebookUrl: string;
+    following: [
```

Comment:
suggestion: if you're only going to store profileId's, you don't have to wrap them in an object.
`following: string[]`, where each item is the profileId